### PR TITLE
add column POI_ID in glossary reports

### PR DESCRIPTION
**Pull request by sanchezmauri**
## Description
In few word, describe what’s your Pull Request about.

### Issue involved
Fixes #[MPRC-6008](https://mercadolibre.atlassian.net/browse/MPRC-6008)

### Checklist:
Before sending your contribution, please specify the following: 

- [ ] This request modifies code snippets. 
- [x] The contribution aligns with the information stated in the repository wiki.

- [ ] Contribution incorporates an article not included until now, which must be added to indexes as a new section. 
- [ ] Contribution includes new features. 
- [ ] New features were communicated in changelog.
- [ ] Requires translations.

### CODE REVIEW

File: guides/manage-account/reports/account-money/glossary.en.md [hunk 1]
```diff
@@ -43,6 +43,7 @@ We know, some terms are technical and you may not be familiar with all of them.
 | SHIPMENT_MODE | Shipping Mode. |
 | PACK_ID | Package identification in the cart. |
 | TAXES_DISAGGREGATED | Taxes disaggregated in JSON format. |
+| POI_ID | POS Serial Number (S/N). |
```
Comment (matiasleonperalta) [inline guides/manage-account/reports/account-money/glossary.en.md:46]: [r] - Esta bien que sea POS Serial Number ? Cual seria el significado de POS ?

**Review by matiasleonperalta — COMMENTED** [review]
La unica duda que tengo es el POS para cuando es ingles.

File: guides/manage-account/reports/account-money/glossary.en.md (hunk 1 above)
Comment (sanchezmauri) [reply guides/manage-account/reports/account-money/glossary.en.md:46]: En los detalles de la columna `POS` tengo las siguientes descripciones:

> | POS_ID | Cash ID if the payment is made through a physical commerce. |
> | POS_NAME | Cash name for the payment made through a physical commerce. |

Esto completa la información o hay que aclararlo?

Comment (matiasleonperalta) [reply guides/manage-account/reports/account-money/glossary.en.md:46]: Claro pero nosotros queremos explicar el POI_ID, que es el point_of_interaction (POINT). El POS, es otra columna aparte.

Comment (sanchezmauri) [reply guides/manage-account/reports/account-money/glossary.en.md]: Ya esta corregido Mati. Al fina quedó:

> | POS_NAME | Point Serial Number (S/N). |

**Review by matiasleonperalta — APPROVED** [review]

**Review by matiasleonperalta — APPROVED** [review]

**Review by elvisjhonataoliveira — APPROVED** [review]

**Review by abderossi — APPROVED** [review]

**Pull request merged**